libr/bin/d/dll/WHLP16T.sdb.txt

```diff
@@ -0,0 +1,10 @@
+1=THK_THUNKDATA16
```

Comment:
all those sdbs are taking a lot of space (about 7MB without counting those). i think we should make the plugin system smarter and associate data files in case we dont want them at build/install time. but that's just a random though on this issue, not aiming to discuss or fix here